src/query/ast/tests/it/testdata/statement.txt

```diff
@@ -13593,6 +13593,74 @@ DescribeTask(
 )
 
 
+---------- Input ----------
+CREATE CONNECTION IF NOT EXISTS my_conn STORAGE_TYPE='s3'
```

Comment:
Would you like to add a test case for CREATE CONNECTION with args like root and endpoint?

Reply:
root is not valid arg for now?
do you mean a valid arg and a invalid or 2 valid args?

Reply:
done